modules/assisted-installer-assisted-installer-prerequisites.adoc

```diff
@@ -0,0 +1,60 @@
+// This is included in the following assemblies:
+//
+// installing-on-prem-assisted.adoc
+:_content-type: CONCEPT
+
+[id='assisted-installer-prerequisites_{context}']
+= Assisted Installer prerequisites
+
+The {ai-full} validates the following prerequisites to ensure successful installation.
+
+== Hardware
+
+For control plane nodes or the {sno} node, nodes must have at least the following resources:
+
+* 8 CPU cores
+* 16.00 GiB RAM
+* 100 GB storage
+* 10ms write speed or less for etcd `wal_fsync_duration_seconds`
+
+For worker nodes, each node must have at least the following resources:
+
+* 4 CPU cores
+* 16.00 GiB RAM
+* 100 GB storage
+
+== Networking
+
+The network must meet the following requirements:
+
+* A DHCP server unless using static IP addressing.
+* A base domain name. You must ensure:
```

Comment:
This needs fixed based on the IBMSG usage of colons. It needs to have a noun, for example,

"You must ensure that the following conditions are met:"

Reply:
Fixed.